components/screens/MenuScreen.tsx

```diff
@@ -1,32 +1,90 @@
-import {useContext} from 'react';
+import React from 'react';
 
 import {StyleSheet} from 'react-native';
 import {SafeAreaView} from 'react-native-safe-area-context';
 
+import * as Updates from 'expo-updates';
+
 import {Heading, HStack, VStack, Switch, Text} from 'native-base';
 
-import {ClientContext, ClientProps} from 'clientContext';
-import {AvalancheCenterSelector} from 'components/AvalancheCenterSelector';
+import {AvalancheCenterCard, AvalancheCenterSelector} from 'components/AvalancheCenterSelector';
+
+import {createNativeStackNavigator, NativeStackScreenProps} from '@react-navigation/native-stack';
 
-export const MenuScreen = () => {
-  const {staging, toggleStaging, setAvalancheCenter} = useContext<ClientProps>(ClientContext);
+import {MenuStackParamList, MenuStackNavigationProps} from 'routes';
+import {useNavigation} from '@react-navigation/native';
+
+const MenuStack = createNativeStackNavigator<MenuStackParamList>();
+export const MenuStackScreen = (
+  center_id: string,
```

Comment:
nitpick, but we should probably default to either camelCase or snake_case

Reply:
Agreed - do you have a preference?

Reply:
I think things are self-consistent in this PR but I can do a pass after this to collapse to a single case in the rest of the codebase